### ISSUE TITLE
Create a bucket on Amazon s3 with Ansible results in Location Constraint Error for region us-east-1

### ISSUE THREAD

**Issue by danieldbower** [ansible/ansible-modules-extras#1200]
I'm trying to repeat the simplest example for creating an s3 bucket for us-east-1 from the documentation, but I keep getting the following error:

An exception occurred during task execution. To see the full traceback, use -vvv. The error was: 
    &lt;Error>
      &lt;Code>InvalidLocationConstraint&lt;/Code>
      &lt;Message>The specified location-constraint is not valid&lt;/Message>
      &lt;LocationConstraint>us-east-1&lt;/LocationConstraint>
      &lt;RequestId>---Redacted---&lt;/RequestId>&lt;HostId>---Redacted---&lt;/HostId>
    &lt;/Error>

Playbook Target:

```
- s3_bucket:
    aws_access_key: "{{aws_access_key}}"
    aws_secret_key: "{{aws_secret_key}}"
    name: "myapp-{{inst}}"
    region: us-east-1
```

I've tried leaving out region altogether, setting it to blank, and putting quotes around it. Each time it fails.

If I change the region to us-west-1, it works fine.

Using Ansible v2.0.0-0.4.beta2

**Comment by ryansydnor** [comment]
This is fixed by https://github.com/ansible/ansible-modules-extras/pull/1268.

**Comment by bcoca** [comment]
closing via merged #1268
